doc/install/prod-ubuntu.md

```diff
@@ -0,0 +1,42 @@
+### Production Installation on Ubuntu 14.04
+
+1. Download [latest stable compiled verison of Mattermost](https://github.com/mattermost/platform/releases) from GitHub 
+2. Set up machine with Ubuntu 14.04 with 2GB of RAM or similar 
+3. Install and configure Ngnix as a proxy to Mattermost 
+4. Install SSL certificate 
+5. Configure proxy pass thru
+6. Install Postgres 9.3+ or MySQL 5.2+ (optionally, this install could be made on another machine)
+7. Create a database using following SQL commands:  
+
+   ```
+   DROP DATABASE mattermost;
```

Comment:
This should probably be `DROP DATABASE IF EXISTS mattermost;` otherwise there will be an error.